docs/devguide/newbeat.asciidoc

```diff
@@ -480,3 +481,35 @@ func main() {
 
 When you're done with your new Beat, how about letting everyone know? Open
 a pull request to add your link to the {beats-ref}/community-beats.html[Community Beats] list.
+
+[[newbeat-migrate-gomodules]]
+=== Migrate existing Beat to go modules
```

Comment:
@radoondas you may be interested on that :slightly_smiling_face:

Reply:
Ou yeah! This is something I will have a look once it is available! Thanks